scripts/e2e-tests.sh

```diff
@@ -171,7 +171,7 @@ function resolve_k8s_version() {
     E2E_CLUSTER_VERSION="${gke_versions[0]}"
     echo "Using latest version, ${E2E_CLUSTER_VERSION}"
   else
-    local latest="$(echo "${gke_versions[@]}" | tr ' ' '\n' | grep -E ^${target_version} | sort -V | tail -1)"
+    local latest="$(echo "${gke_versions[@]}" | tr ' ' '\n' | grep -E ^"${target_version}" | sort -V | tail -1)"
```

Comment:
this would be better with the `"` before the `^` instead of just after it

Reply:
done